packages/create-react-admin/src/app.tsx

```diff
@@ -58,10 +58,6 @@ const stepReducer = (
             const newState = {
                 ...state,
                 dataProvider: action.value,
-                resources:
-                    action.value === 'ra-data-fakerest'
-                        ? ['posts', 'comments']
-                        : state.resources,
```

Comment:
regression: if I don't enter the resources, 'posts' and 'comments' are no longer used automatically